app/views/_user_util_links.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="navbar-right">
+    <ul class="nav navbar-nav">
+        <%= render_nav_actions do |config, action|%>
+            <li class="nav-item"><%= action %></li>
+        <% end %>
+
+        <% if has_user_authentication_provider? %>
+            <% if current_user %>
+                <li class="nav-item">
+                    <%= link_to t('blacklight.header_links.logout'), destroy_user_session_path, class: 'nav-link' %>
+                </li>
+                <% unless current_user.to_s.blank? -%>
+                    <li class="nav-item">
+                        <%= link_to current_user.name, root_path, class: 'nav-link' %>
+                    </li>
+                <% end %>
+            <% else %>
+                <li class="nav-item">
+                    <%= link_to "Log in", "/users/auth/google_oauth2", class: 'nav-link' %>
```

Comment:
we can use `link_to t('blacklight.header_links.login')` to allow translation, since it's already defined